### PR TITLE
Users/alexmeng/chat interface component

### PR DESCRIPTION
**Pull request by ZhuoyangM**
### Summary
add a brief summary of the code change

### Screenshot
upload the screenshot of the changes made

### Checklist
- [x] My code follows ESLint rules and passes all tests.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: insta-ui/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "next": "14.2.13",
     "react": "^18",
     "react-dom": "^18",
+    "react-icons": "^5.3.0",
```
Comment (junkaiman) [inline insta-ui/package.json]: we've imported the font-awesome so you can use that one to reduce bundle size!

File: insta-ui/src/components/ChatInterface/InputBar.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { AiOutlinePaperClip, AiOutlineSend } from "react-icons/ai";
```
Comment (junkaiman) [inline insta-ui/src/components/ChatInterface/InputBar.tsx]: You can take a look at this reference [[ref]](https://github.com/junkaiman/ui-generator/blob/7d9601a94f0d243f0f8319ec434732e9c9150fe4/insta-ui/src/components/SideBar/SideBarItem.tsx#L2C1-L5C1) to replace with font-awesome icon

File: insta-ui/package.json (hunk 1 above)
Comment (ZhuoyangM) [reply insta-ui/package.json]: Resolved

File: insta-ui/src/components/ChatInterface/InputBar.tsx (hunk 2 above)
Comment (ZhuoyangM) [reply insta-ui/src/components/ChatInterface/InputBar.tsx]: resolved

File: insta-ui/src/app/c/[slug]/page.tsx
Comment (junkaiman) [inline insta-ui/src/app/c/[slug]/page.tsx:1]: Code change might conflict with `Previewer` @lpplake . pls help resolve conflict when necessary
https://github.com/junkaiman/ui-generator/pull/22/commits/ae91333d86d4e47a991449287e88c1289e4a4a4f

**Review by junkaiman — APPROVED** [review]

**Pull request merged**